package.json

```diff
@@ -89,6 +89,7 @@
 		"@types/jest": "^28.1.8",
 		"@typescript-eslint/eslint-plugin": "^5.35.1",
 		"@typescript-eslint/parser": "^5.35.1",
+		"app-root-path": "^3.1.0",
```

Comment:
This should be marked under `dependencies` and not a `devDependencies`

Reply:
same for chalk.

Reply:
ah yeah, correct.